src/components/organisms/UiList/_internal/UiListItemSuffix.vue

```diff
@@ -60,7 +60,7 @@ export interface ListItemSuffixProps {
   /**
    * Use this props to set list item suffix tag.
    */
-  tag?: HTMLTag;
+  tag?: string | HTMLTag;
```

Comment:
Isn't HTMLTag a narrowed string type already? Why this change? This type doesn't make sense because the `string` type already includes `HTMLTag`. I see the reason - because of the default `''` value - but IMO, it's not a correct solution. Its default value should be undefined (or null; it depends), but not `''`.